packages/twenty-front/src/modules/views/utils/mapGroupDefinitionsToViewGroups.ts

```diff
@@ -0,0 +1,18 @@
+import { RecordGroupDefinition } from '@/object-record/record-group/types/RecordGroupDefinition';
+import { ViewGroup } from '@/views/types/ViewGroup';
+
+export const mapGroupDefinitionsToViewGroups = (
```

Comment:
Please be very precise with the naming : `mapRecordGroupDefinitionsToViewGroups`, otherwise it makes it easy to be misled when reading complex parts of the codebase like this.